src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double width;
+    private final double length;
+
+    public Rectangle(double width, double length) {
+        this.width = width;
+        this.length = length;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
```

Comment:
The method name `toDraw` does not follow Java naming conventions for methods. It should be a verb, such as `draw` to clearly represent the action it performs.